dotfiles-setup.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
```

Comment:
this is copied directly from setup.sh. I did this so I could copy over the dotfiles before running an os-specific install, so [this](https://github.com/Khan/khan-dotfiles/pull/99/files#diff-db35c16b3f04fa41c3d18e63f39627bb1bfb84b52b422ff4f3328ea76e511b66R110) could work.